editoast/Cargo.toml

```diff
@@ -164,3 +164,10 @@ tempfile.workspace = true
 
 [lints]
 workspace = true
+
+[profile.dev]
+debug = "none"
```

Comment:
Contrary to your last comment, `debug` is still set to `none` instead of `limited`.

Reply:
Did I forgot to push? and the response is "yes" 🤦